templates/patterns/copyright_overlay/pattern-copyright-overlay.html.twig

```diff
@@ -0,0 +1,57 @@
+{#
+/**
+ * @file
+ * Copyright overlay pattern.
+ */
+#}
+{{ attach_library('oe_bootstrap_theme/copy_clipboard') }}
+
+{% set _trigger_label = trigger_label|default('View more'|t) %}
+
+{% set _classes = ['copyright-overlay', 'px-2', 'text-muted'] %}
+
+{% set _footer %}
+  {% include '@oe-bcl/button' with {
+    label: 'Copy Copyright'|t,
+    attributes:  create_attribute()
+      .setAttribute('data-copy-target', 'copyright-text'),
+  } only %}
+{% endset %}
+
+{% set _header %}
+  <h5 class="mb-0">{{ 'Copyright'|t }}</h5>
+  {% include '@oe-bcl/button' with {
+    clean_class: true,
+    attributes:  create_attribute()
+      .addClass(['btn-close'])
+      .setAttribute('data-bs-dismiss', 'modal')
+      .setAttribute('aria-label', 'Close'),
+  } only %}
+{% endset %}
+
+{% set _body %}
+  <div class="copyright-text">
```

Comment:
cant remember how we named it in the card v2, but I feel it should be body or content and not "text" as it can be html/render array

Reply:
done, changed it to content.